ui/src/hooks/useFileManage.ts

```diff
@@ -686,14 +702,16 @@ export const useFileManage = (token: string, t: any) => {
         uploadFileList,
         onFinish,
         onError,
-        onProgress
+        onProgress,
+        loadingMessage
       }: {
         uploadFileList: Ref<Array<UploadFileInfo>>;
         onFinish: () => void;
         onError: () => void;
         onProgress: (e: { percent: number }) => void;
+        loadingMessage?: any;
       }) => {
-        handleFileUpload(<WebSocket>socket, uploadFileList, onProgress, onFinish, onError, t);
+        handleFileUpload(<WebSocket>socket, uploadFileList, onProgress, onFinish, onError, t, loadingMessage);
       }
     );
 
```

Comment:
The provided code seems to be incomplete and there seem to be inconsistencies between importing namespaces ('mittBus', 'type', `FileManage`) which differ across the import section. Additionally, it's not clear how this module was intended to work.

Without more context or specific questions about certain sections of the code, I can't offer detailed analyses for potential issues or optimizations. But generally, reviewing imports, making sure they match across modules and understanding their purpose can help improve code quality and readability.